src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -910,12 +932,12 @@
         - name: Create a load balancer on a specific virtual network and subnet.
           text: >
             az network lb create -g MyResourceGroup -n MyLb --vnet-name MyVnet --subnet MySubnet
-        - name: create a zone flavored public facing load balancer through provisioning a zonal public ip
+        - name: Create a zone flavored public facing load balancer through provisioning a zonal public ip.
           text: >
-            az network lb create -g MyResourceGroup -n myLB --public-ip-zone 2
-        - name: create a zone flavored internal facing load balancer through provisioning a zonal frontend ip configuration
+            az network lb create -g MyResourceGroup -n MyLB --public-ip-zone 2
+        - name: create a zone flavored internal facing load balancer through provisioning a zonal frontend ip configuration.
```

Comment:
Summaries of examples must be capitalized, complete sentences.

Reply:
Fixed on my branch.